.github/scripts/install.sh

```diff
@@ -58,17 +58,7 @@ else
 fi
 
 # Install subrepos in development mode
-for dep in $(ls external-deps)
-do
-    echo "Installing $dep subrepo"
-
-    # This is necessary to pass our minimal required version of PyLSP to setuptools-scm
-    if [ "$dep" = "python-lsp-server" ]; then
-        SETUPTOOLS_SCM_PRETEND_VERSION=`python pylsp_utils.py` pip install --no-deps -q -e external-deps/$dep
-    else
-        pip install --no-deps -q -e external-deps/$dep
-    fi
-done
+python install_local.py
```

Comment:
```suggestion
python -bb -X dev -W error install_local.py
```

Best practice, to report warnings and more informative behavior in case of issues.

Reply:
I can't reach lines 68-72, but is the following necessary if we already install spyder locally in develop mode?
```bash
# Install Spyder to test it as if it was properly installed.
# Note: `python setup.py egg_info` doesn't work here but it
# does locally.
python setup.py -q bdist_wheel
pip install --no-deps -q dist/spyder*.whl
```

Reply:
No, but if we do need to do it for some reason, the first line is deprecated; instead, it should be

```bash
python -m build
```

or more robustly,

```bash
python -bb -X dev -W error -m build
```

Of course, ``build`` needs to be installed.